cypress/integration/page.spec.js

```diff
@@ -56,9 +56,9 @@ Cypress.on('fail', (e) => {
 });
 
 describe('', () => {
-  beforeEach(() => {
-    if (failed) Cypress.runner.stop();
-  });
+  // beforeEach(() => {
+  //   if (failed) Cypress.runner.stop();
+  // });
 
```

Comment:
why you comment this?

Reply:
i commented this because if one test fails tests stop